yang/frr-zebra.yang

```diff
@@ -1845,6 +1905,38 @@ module frr-zebra {
           "Link bandwidth informational parameter, in megabits.";
       }
       // TODO -- link-params for (experimental/partial TE use in IGP extensions)
+      container state {
+        config false;
+        description
+          "Operational data.";
+        leaf zif-type {
+          type identityref {
+            base zebra-interface-type;
+          }
+        }
+
+        leaf vlan-id {
```

Comment:
vlan-id read only state?

Reply:
This is under zebra's state container in read-only field.

Reply:
No I mean should it be read only or is this something that can be configured in future from FRR CLI?

Reply:
No plan as of now. We will keep it read-only field.